Rocket-Nozzle-Design-and-Analysis/Rocket_Nozzle_Design_and_Analysis.tex

```diff
@@ -0,0 +1,235 @@
+% Below are examples of using nomenclature for math symbols and constants or units
+% \nomenclature{$\dot{m}$}{Mass flow rate
+%   \nomunit{\,\si{\kilo\gram\per\second}}}
+% \nomenclature{$c$}{Speed of light
+%  \nomunit{\,\SI{2.9979e8}{\meter\per\second}}}
+\printnomenclature{}
+
+\section{Introduction}
+\label{sec:introduction}
+\IEEEPARstart{P}{ropulsion} systems are very difficult to design and test. The SPEX propulsion team has worked for two semesters, Spring 2017 & Fall 2017, to design and test a cold gas propulsion system.
+There was moderate success in the design and testing of this system. The successes included, nozzle design, nozzle fabrication, test stand fabrication, and experimentation.
+The nozzles were designed through the Aerospike software given to us by an advisor, Dr. Dorin Patru. The nozzles were fabricated by methods of additive manufacturing.
+The test stand was properly designed and fabricated to work with thrust levels up to 10 Newtons. The experimentation was successful because
+the team was able to test multiple nozzles with different operating characteristics. There developed a problem in the fabrication of conical nozzles that the team designed using the analytical rocket equations.
+The nozzles for the thrust levels were too small. The fabrication process would be inhibitively difficult. The solution to this is an increase in the thrust level.
+ This project will focus on the design of a conical nozzle for a 100 Newton thrust force test.
+
+\section{Primary Objective}\label{sec:primary-obj}
+The primary goal of this project is to use the analytical rocket equations to design a conical rocket nozzle to produce 100 Newtons of thrust.
+The team will design a new stand that is capable of securely fastening a propulsion test of 200 Newtons.
+Thus providing a factor of safety of 2.0. The tests will be performed with Air or Nitrogen so the extensive tabulated data can be used in the performance analysis of the nozzle.
+
+\section{Benefit to SPEX}\label{sec:benefit}
+The benefit to SPEX would be the further education in the design of rocket nozzles using the analytical rocket equations. The experiment platform would
+be a great exhibit for visiting companies or Imagine RIT where SPEX would be on display. This project would be
+the first time that the propulsion team has fully designed and operated a rocket nozzle. Using these equations correctly is very complicated and important for
+propulsion engineers to understand. The project would have a full life cycle which includes design, manufacture, test, and reporting the results.
+
+\section{Implementation}\label{sec:implementation}
+The project will start with a brief eduation of the rocket equations. The team will go through each one so everyone on the team is acquainted with them before we dive into the
+nozzle design. Then the team will start with the nozzle design. Once the nozzle design is complete the team will design the test stand that can adequately contain the
```

Comment:
There is unneeded repetition of the noun 'nozzle design' across 3 adjacent sentences.

Also the nozzle design process is not defined in any way. It could use some explanation, or some determination of the parameters being sought during the design process. Such as dimensions, materials, exhaust velocity, expansion ratio, minimum throat diameter, etc.